python/graphstorm/trainer/ep_trainer.py

```diff
@@ -84,43 +88,59 @@ def fit(self, train_loader, num_epochs,
             freeze_input_layer_epochs=0,
             max_grad_norm=None,
             grad_norm_type=2.0):
-        """ The fit function for edge prediction.
+        """ Fit function for edge prediction.
+
+        This function performs the training for the given edge prediction model.
+        It iterates over the training batches provided by the ``train_loader``
+        to compute the loss, and then performs the backwards step using trainer's
+        own optimizer. 
+
+        If an evaluator and a validation dataloader are added to this trainer, during
+        training, the trainer will perform model evaluation in three cases:
 
-        Performs the training for `self.model`. Iterates over the training
-        batches in `train_loader` to compute the loss and perform the backwards
-        step using `self.optimizer`. If an evaluator has been assigned to the
-        trainer, it will run evaluation at the end of every epoch.
+        * At the end of each epoch.
+        * At the evaluation frequency (number of iterations) defined in the evaluator.
+        * Before saving a model checkpoint.
 
         Parameters
         ----------
-        train_loader : GSgnnEdgeDataLoader
-            The mini-batch sampler for training.
-        num_epochs : int
-            The max number of epochs to train the model.
-        val_loader : GSgnnEdgeDataLoader
-            The mini-batch sampler for computing validation scores. The validation scores
-            are used for selecting models.
-        test_loader : GSgnnEdgeDataLoader
-            The mini-batch sampler for computing test scores.
-        use_mini_batch_infer : bool
-            Whether or not to use mini-batch inference.
-        save_model_path : str
-            The path where the model is saved.
-        save_model_frequency : int
-            The number of iteration to train the model before saving the model. Default is -1,
-            meaning only save model after each epoch.
-        save_perf_results_path : str
-            The path of the file where the performance results are saved.
+        train_loader: GSgnnEdgeDataLoader
+            Edge dataloader for mini-batch sampling the training set and train the model.
+        num_epochs: int
+            The max number of epochs used to train the model.
+        val_loader: GSgnnEdgeDataLoader
+            Edge dataloader for mini-batch sampling the validation set and computing
```

Comment:
The dataloader only sample mini-batches. It does not compute scores.

Reply:
Remove this description in all occurrences.